GenerambaTemplates/viper-code-layout/Code/viewlayout.swift.liquid

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+final class {{ module_info.name }}ViewLayout: UIView {
```

Comment:
We have setupLayout() and configure() methods in each(or almost) ViewLayout class. So I think we can add it to private extension and remove if we for some reasons don't need them for current implementation